src/components/DisplayEntries.tsx

```diff
@@ -37,14 +45,8 @@ class DisplayEntries extends React.Component<DisplayEntriesProps, any> {
     }
 }
 
-interface DisplayEntriesPropsTypes {
-    salaries: number[];
-}
-
 const mapStateToProps = (state: AppState) => ({
-    /* salaries references the rootReducer index, salaryArray references the
-    array defined in salaryReducer     */
-    salaries: state.salaries.salaryArray
+    employees: state.employees.employeeArray
```

Comment:
the variable name `state.employees.employeeArray` is using a form of Hungarian notation, which is considered deprecated  in modern programming languages (the type is inferred from the type system, which leads to duplication when it's also declared in the variable name) 

The name `employees` should be enough for the reader to infer that the variable is a collection